src/e2e/pages/landingPage.ts

```diff
@@ -43,7 +43,7 @@ export class LandingPage {
     this.MozillaFooterLogoLink = page.locator("footer a").first();
     this.AllBreachesLink = page.getByRole("link", { name: "All Breaches" });
     this.FAQLink = page.locator('footer a:has-text("faq")');
-    this.TermsLink = page.getByRole("link", { name: "Terms & Privacy" });
+    this.TermsLink = page.getByRole("link", { name: "Terms of Service" });
```

Comment:
Unrelated to this PR, but now that we have separate "Terms of Service" and "Privacy Notice" footer links, do we have/need a test for also checking the Privacy Notice footer link? https://www.mozilla.org/privacy/subscription-services?

Also, probably not worth fixing, but there seems to be some inconsistency w/ naming between stuff like "ourMissionLink" and "TermsLink" where sometimes props have initial uppercase and sometimes initial lowercase (PascalCase vs camelCase).

Reply:
good point here, will update